gammapy/astro/population/spatial.py

```diff
@@ -189,8 +188,8 @@ def evaluate(r, amplitude, a, b):
 
 
 class FaucherKaspi2006(Fittable1DModel):
-    r"""Radial distribution of the birth surface density of pulsars in the galaxy
-     - Faucher-Giguere & Kaspi 2006.
+    r"""
+    Radial distribution of the birth surface density of pulsars in the galaxy - Faucher-Giguere & Kaspi 2006.
```

Comment:
Should we start using a `References` section at the end of docstrings?

Reply:
Could be a good idea!